enterprise/backend/test/metabase_enterprise/semantic_search/index_embedding_test.clj

```diff
@@ -5,21 +5,3 @@
    [metabase-enterprise.semantic-search.embedding :as semantic.embedding]
    [metabase-enterprise.semantic-search.index :as semantic.index]
    [metabase.test :as mt]))
-
-(deftest ^:parallel index-embedding-name-length-test
```

Comment:
personally I still think there's value in having some kind of tests that checks that none of the names exceed the limit, and that all the names are unique, even if it means maintaining a list of known models / providers from ai-service that has to be manually kept in sync. otherwise, it's just too easy to add a new model / provider / index and not realize the name is getting truncated or collides with another name.

Reply:
will address these comments tomorrow 👍